spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/PoolControllerHelperService.java

```diff
@@ -745,6 +745,12 @@ public JSONObject getPoolableElementInfo(HttpSession session, JSONObject json) {
         Long elementId = json.getLong("elementId");
         Pool<? extends Poolable> pool = requestManager.getPoolById(poolId);
         StringBuilder info = new StringBuilder();
+        double total = 0;
+        for (Poolable p : pool.getPoolableElements()) {
+          if (p instanceof Dilution) {
+            total += ((Dilution) p).getConcentration();
```

Comment:
The total concentration depends on volume as well (otherwise you end up with 100 ng in one uL), which doesn't get captured by this equation :)